src/longRunningCalls/longrunning.ts

```diff
@@ -129,13 +160,14 @@ export class Operation extends EventEmitter {
    * request.
    */
   cancel() {
-    if (this.currentCallPromise_) {
-      this.currentCallPromise_.cancel();
+    if (!this.diregapic && this.currentCallPromise_) {
```

Comment:
I believe there is no cancellation for diregapic. Can we just throw?
```ts
if (this.diregapic) { 
  throw new Error("Cancellation is not supported for this method");
}
```
@vam-google what do you think?